internal/mesh/internal/controllers/sidecarproxy/builder/destination_multiport_test.go

```diff
@@ -41,6 +41,31 @@ func TestBuildMultiportImplicitDestinations(t *testing.T) {
 		},
 	}
 
+	multiportServiceData := &pbcatalog.Service{
+		Ports: []*pbcatalog.ServicePort{
```

Comment:
setting up service data local to the test because sharing the service data with all of the other tests was causing lots of conflicts and coordination.